app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -229,7 +252,9 @@ public function getWebsites($withDefault = false, $codeKey = false)
     }
 
     /**
-     * {@inheritdoc}
+     * Reinitialize store list
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc